controller/src/beerocks/bml/internal/bml_internal.cpp

```diff
@@ -3282,4 +3527,4 @@ bool bml_internal::wake_up(uint8_t action_opcode, int value)
     }
     m_prmCliResponses[action_opcode]->set_value(value);
     return true;
-}
+}
```

Comment:
need to add empty new line at end of file

Reply:
Added